src/Microsoft.Performance.Toolkit.Engine/ITableResult.cs

```diff
@@ -28,6 +29,14 @@ public interface ITableResult
         /// </summary>
         IReadOnlyCollection<IDataColumn> Columns { get; }
 
+        /// <summary>
+        ///     Gets the mapping of columns that have been added to the table with variants to their added variants.
```

Comment:
>Gets the mapping of columns that have been added to the table with variants to their added variants. [](http://example.com/codeflow?start=16&length=100)

this wording is a bit confusing

Reply:
Updating to `Gets the mapping from columns with variants to those variants.`